.github/workflows/deploy.yml

```
@@ -22,14 +22,14 @@ jobs:
         uses: actions/checkout@v4
         
       - name: Setup Python
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@v5
         with:
-          python-version: '3.11'
+          python-version: '3.12'
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)